acceptance-tests/src/test/java/tech/pegasys/signers/keystorage/hashicorp/tests/HashicorpVaultAccessAcceptanceTest.java

```diff
@@ -48,11 +52,16 @@ void cleanup() {
       hashicorpNode.shutdown();
       hashicorpNode = null;
     }
+
+    try {
+      docker.close();
+    } catch (final Exception ignored) {
```

Comment:
nit: may be put a system.err or debug log that docker close failed. Empty blocks are frowned upon by some devs :)

Reply:
yeah -fixed.